zabbix_auto_config/processing.py

```diff
@@ -225,7 +225,11 @@ def __init__(self, name, state, db_uri, host_modifier_dir):
     def get_host_modifiers(self):
         sys.path.append(self.host_modifier_dir)
 
-        module_names = [filename[:-3] for filename in os.listdir(self.host_modifier_dir) if filename.endswith(".py")]
+        try:
+            module_names = [filename[:-3] for filename in os.listdir(self.host_modifier_dir) if filename.endswith(".py")]
+        except(FileNotFoundError):
```

Comment:
Parenthesis could be considered a linting error/warning. [C0325](https://pylint.pycqa.org/en/latest/messages/convention/superfluous-parens.html).